src/mark.js

```diff
@@ -61,6 +61,7 @@ class Mark { // eslint-disable-line no-unused-vars
             "caseSensitive": false,
             "ignoreJoiners": false,
             "ignoreGroups": 0,
+            "invalidMax": true,
```

Comment:
Or, should we just remove it completely and have values larger than the content automatically set to max?